nbs/03_export2html.ipynb

```diff
@@ -1155,6 +1155,80 @@
     "    return pnb"
```

Comment:
<p>Please keep the doc comment to one line between double quotes. Any additional text can go in a Markdown cell just after and will be shown in the docs.</p><p>Also <code>_rep_citelink</code>  doesn't seem to be used anywhere?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>